.github/labeler-issue-triage.yml

```
@@ -296,6 +296,8 @@ service/route53:
   - '((\*|-) ?`?|(data|resource) "?)aws_route53_(?!resolver_)'
 service/route53domains:
   - '((\*|-) ?`?|(data|resource) "?)aws_route53domains_'
+service/route53recoverycontrolconfig:
+  - '((\*|-) ?`?|(data|resource) "?)aws_route53_recovery_control_config_'  
```

We prefer the service part of the resource type name to be a single word (i.e. no `-` or `_` characters) - this will aid with future automation.

```suggestion
  - '((\*|-) ?`?|(data|resource) "?)aws_route53recoverycontrolconfig_'  
```

---

sure. fixed it